src/sql/workbench/contrib/assessment/browser/media/asmt.css

```diff
@@ -0,0 +1,248 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+asmtview-component {
+	height: 100%;
+	width: 100%;
+	display: block;
+}
+
+.asmt-heading {
+	font-weight: bold;
+	font-size: large;
+}
+
+#asmtViewDiv .propertiesSectionTitle {
+	margin-bottom: 20px;
+	font-size: larger;
+}
+
+#asmtViewDiv .propertyBlock {
+	display: inline-block;
+	margin: 0px 0px 20px 20px;
+}
+
+#asmtViewDiv .propertyLabel {
+	font-size: 11px;
+}
+
+.asmt-heading-container {
+	height: 30px;
+	border-bottom: 3px solid #f4f4f4;
```

Comment:
Are there appropriately colored in the light and dark default themes? In general it's better to use the theme service and hook up to a common color - such as https://github.com/microsoft/azuredatastudio/blob/master/src/vs/workbench/common/theme.ts#L262

This comment applies to all the colors in this file.

Reply:
I have replaced all the colors with the common ones